ndk/build.gradle.kts

```diff
@@ -0,0 +1,209 @@
+import com.diffplug.spotless.LineEnding
+import com.vanniktech.maven.publish.MavenPublishBaseExtension
+import com.vanniktech.maven.publish.MavenPublishPlugin
+import com.vanniktech.maven.publish.MavenPublishPluginExtension
+import groovy.util.Node
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+    `java-library`
+    id("com.diffplug.spotless") version "6.11.0" apply true
+    id("io.gitlab.arturbosch.detekt") version "1.19.0"
+    `maven-publish`
+    id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.13.0"
+
+}
+
+buildscript {
+    repositories {
+        google()
+    }
+    dependencies {
+        classpath("com.android.tools.build:gradle:7.4.2")
+        classpath(kotlin("gradle-plugin", version = "1.8.0"))
+        classpath("com.vanniktech:gradle-maven-publish-plugin:0.18.0")
+        // dokka is required by gradle-maven-publish-plugin.
+        classpath("org.jetbrains.dokka:dokka-gradle-plugin:1.7.10")
+        classpath("net.ltgt.gradle:gradle-errorprone-plugin:3.0.1")
+
+        // legacy pre-prefab support
+        // https://github.com/howardpang/androidNativeBundle
+        classpath("io.github.howardpang:androidNativeBundle:1.1.4")
```

Comment:
Just to be clear, we keep this for compatibility. This is for users who build their own `sentry-android` so that their build does not break, right?

Reply:
Yes, this is just for compatibility reasons, intended for app developers who have native (C/C++) code in their Android apps. That's also what our documentation currently recommends: https://docs.sentry.io/platforms/android/configuration/using-ndk/#allowing-the-compiler-to-link-libraries